app/models/merchant.rb

```diff
@@ -14,20 +14,7 @@ def top_five_customers
       .limit(5)
   end
 
-end
-
-# SELECT merchants.id AS merchant_id, merchants.name AS merchants_name, customers.id AS customer_id, customers.first_name, customers.last_name, count(transactions.result) AS transaction_result
-
-# FROM customers
-# JOIN invoices ON invoices.customer_id = customers.id
-# JOIN transactions ON transactions.invoice_id = invoices.id
-# JOIN invoice_items ON invoice_items.invoice_id = invoices.id
-# JOIN items ON items.id = invoice_items.item_id
-# JOIN merchants ON merchants.id = items.merchant_id
-
-# WHERE transactions.result = '1' AND merchants.id = '1'
-
-# GROUP BY merchants.id, customers.id
-
-# ORDER BY merchants.id, transaction_result DESC
-# LIMIT 5;
\ No newline at end of file
+  def rts_items #ready_to_ship
```

Comment:
I think this should be ready_to_ship_items

Reply:
I agree for developer empathy!